README.md

```diff
@@ -316,6 +316,11 @@ OPTIONS:
    --s3.secret_access_key value The S3/minio secret access key to use when
       using S3 proxy backend. Applies to s3 auth method(s): access_key.
       [$BAZEL_REMOTE_S3_SECRET_ACCESS_KEY]
+      
+   --s3.disable_v4_sign Whether to disable v4 sign and enable v2 sign when 
+      using the S3 proxy backend. Only effect when use access_key as 
+      auth_method. (default: false, is enable v4 sign) 
+      [$BAZEL_REMOTE_S3_DISABLE_V4_SIGN]
```

Comment:
I think it's reasonable to add a configuration option to specify the AWS signature type, but it should be something that is extensible over time for example in case S3 adds a new AWS signature versions.

So instead of adding a `--s3.disable_v4_sign` flag which means "use v2", maybe we could add a `--s3.signature_version` flag which defaults to `"v4"`, but also supports `"v2"` ?

Reply:
That's a good idea. I've sumitted a new commit in which I replaced disable_v4_sign with signature_type. It's just the same as minio's SignatureType.